stresstests/src/test/java/org/neo4j/kernel/stresstests/TransactionThroughputChecker.java

```diff
@@ -32,12 +32,12 @@
     private final List<Double> reports = new ArrayList<>();
 
     @Override
-    public void report( long transactions, long timeSlotMillis, long timeElapsedMillis )
+    public void report( long transactions, long elapsedTime)
     {
-        reports.add( ((double) transactions / (double) timeSlotMillis) );
+        reports.add( ((double) transactions / (double) elapsedTime) );
     }
 
-    public void assertUniformThroughput( PrintStream out )
+    public void assertThroughput( PrintStream out )
```

Comment:
I not really sure that we have normal distribution over there. That we can clearly see in the beginning when throughput is lower then all the rest. Plus final assumption to have 80% of measurements in a average +- range sounds more like normal distribution. Thats why word `uniform` was removed.